src/App.tsx

```diff
@@ -1,60 +1,121 @@
 import './App.scss';
+import React, { useState } from 'react';
+import users from './api/users';
+import importedTodos from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function findBiggestTodoId(todoList: Todo[]): number {
+  return todoList.reduce((prev, todo) => (prev > todo.id ? prev : todo.id), 0);
+}
```

Comment:
i'd get a nextId right away
```suggestion
function getNewTodoId(todoList: Todo[]): number {
  return todoList
  	.reduce((prev, todo) => (
  	  prev > todo.id
  		  ? prev
  		  : todo.id)
  		, 0
  	) + 1;
}
```